Documentation/Guide/Menu/_menu.typoscript

```diff
@@ -0,0 +1,13 @@
+page = PAGE
+page {
+  10 = FLUIDTEMPLATE
+  10 {
+    templateName = Default
+    templateRootPaths.10 = EXT:site_package/Resources/Private/Templates/Pages/
+    # ...
+    dataProcessing {
+      10 = menu
+      as = myMainMenu
+    }
```

Comment:
NOt tested, but I'm pretty sure that will not work.

Reply:
Good catch, where did the 10 go